scene/2d/light_occluder_2d.cpp

```diff
@@ -245,15 +245,22 @@ int LightOccluder2D::get_occluder_light_mask() const {
 }
 
 String LightOccluder2D::get_configuration_warning() const {
+	String warning = Node2D::get_configuration_warning();
+
 	if (!occluder_polygon.is_valid()) {
-		return TTR("An occluder polygon must be set (or drawn) for this occluder to take effect.");
+		if (!warning.empty())
+			warning += "\n\n";
+		warning += TTR("An occluder polygon must be set (or drawn) for this occluder to take effect.");
+		return warning;
```

Comment:
While this is correct in the current logic, there might be further warnings added that don't depend on the occluder being valid, so I wouldn't do an early return here. The result should be the same nevertheless as the next check will evaluate to false.

Reply:
Changed it in the listed files but also found the similar logic in 11 other. So that should be also changed in all other files?

Reply:
Yeah I only commented on a subset but it encompasses all similar patterns.

Reply:
I guess now that should be all of the files updated